src/screens/Profile/Notification.js

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import { View, StyleSheet, AsyncStorage } from 'react-native';
+import { Card, Button, Paragraph } from 'react-native-paper';
+import auth from '@react-native-firebase/auth';
+import Swipeable from 'react-native-gesture-handler/Swipeable';
+import PropTypes from 'prop-types';
+
+export default class Notification extends React.Component {
+  constructor(props) {
+    super(props);
+    const { viewed } = this.props;
+    this.state = { viewed, visible: true };
```

Comment:
Don't need to set `viewed` in the state when you already have it as a prop

Reply:
For me: I used viewed as a state though so don't I have to initialize the state in the constructor like so?

Reply:
You can either
- let the parent do all the state management for the child.
- let the child do its own state management.

If you try to manage the same state in both parent and child, you can easily encounter bugs since you as the developer don't know which component has the correct value of `viewed`. In other words, for the `viewed` state, you only want one source of truth.

Would recommend doing the first approach (which you are already doing) and remove of the `viewed` state in this component.